src/bluefinClient.ts

```diff
@@ -859,6 +859,17 @@ export class BluefinClient {
     );
   };
 
+  /**
+   * @param walletAddress wallet address of the user
+   * @returns string
+   * @description
```

Comment:
description goes at the top of the comment

Reply:
I see in all the newly added methods the `description` comment is at the bottom. Please remedy it. Thanks

Reply:
done :rocket: